README.md

```diff
@@ -19,6 +19,12 @@ Each SDK also contains advanced features in the form of plugins, which extend th
 - {@link placeholder|placeholder} - Used to display a lightweight version of an image while the target image is downloading.
 - {@link responsive|responsive} - Used to resize your images automatically based on the viewport size.
 
+## Development setup
+To build and link project: 
+- clone project
+- npm run quickstart
```

Comment:
Please add a description of what the quickstart script does (so that people who don't know lerna can understand)